### DISCUSSION TITLE
Elixir implementation

### DISCUSSION THREAD

**Discussion by rafaelfess** [connectrpc/.github#18] [Polls]
Would you or your company use a Connect implementation in Elixir? If yes, please leave a comment and tell us more about what you'd like: synchronous or asynchronous, important framework integrations (Phoenix?), and anything else important.

**Comment by perezd** [comment]
Shared to reddit :) https://www.reddit.com/r/elixir/comments/1cxldz7/poll_would_you_or_your_company_use_a_connect_rpc/

**Comment by btkostner** [comment]
I’d be extremely interested in this, and would push my current company to adopt it. Being able to integrate it into an existing “plug” application without having to spin up another web server or use another port would also make it much easier to use.

**Comment by Aetherus** [comment]
Sum types are essential to our projects, so no. I'd rather use a strong and dynamically typed data serialization format, like MessagePack.
